### DISCUSSION TITLE
Cloud offer sunsetting

### DISCUSSION THREAD

**Discussion by gabrielcnegre** [fief-dev/fief#332] [Q&A] [answered]
Hello @frankie567 ! 

I love Fief and ended up using i as part of a commercial project, I am very worried about the cloud offer sunsetting, specifically when will it happen? 

I need time to move my users to another solution, please, could you give what would be the worst scenario? (The earliest you are planning to sunset the cloud offer)

**Comment by frankie567** [comment, marked as answer]
Hi @gabrielcnegre 👋 

Thank you for your trust! TBH, I'm quite open regarding the time frame. Ideally, I would like to settle things before July 2024. But if that doesn't work, well, I can wait more.

In any case, I'll provide guidance and help to extract your data and setup a Fief instance on another hosting provider 🙂 So, in any case, no rush ;)

**Reply by gabrielcnegre** [reply]
Thank you @frankie567 !

Just by having a few more months really helps a lot. I hope we can continue using fief in the future.

**Reply by gabrielcnegre** [reply]
Hey @frankie567 just wanted to ask you, if is it possible towait until the end of july to close fief cloud?

**Reply by frankie567** [reply]
Yes, if it's necessary for you, no problem. Look: I'm on holiday from end of July to early August. So, we can agree on the **12th of August 2024** for Fief Cloud shutdown. Does that work for you?

**Reply by gabrielcnegre** [reply]
Yes! Thank you so much, I really hope I can repay helping with fief eventually.

Again, thank you for extending the time!
